practice_hash.rb

```diff
@@ -0,0 +1,39 @@
+#Let's create a rainbow of colors as an array of hashes, where each hash contains the name of the color (key) and the RGB values (value).
+
+rainbow = [
+  :red => {
+    r: 255,
+    g: 0,
+    b: 0
+  }
+  :orange => {
+    r: 255,
+    g: 125,
+    b: 0
+  }
+
+  :yellow => {
+    r: 255,
+    g: 255,
+    b: 0
+  }
+
+  :green => {
+    r: 0,
+    g: 255,
+    b: 0
+  }
+
+  :blue => {
+    r: 0,
+    g: 0,
+    b: 255
+  }
+
+  :purple => {
+    r: 125,
+    g: 0,
+    b: 255
+  }
+
+]
```

Comment:
You are on the right track here. You're missing some commas separating each element in the array though. And some {}'s making each element a hash inside the array.

Doing the keys was tricky, but I wanted you all to attempt.

It's ok not to have the answer perfect, so thanks for at least attempting! We will go over it in class and if that explanation doesn't make sense, I'm happy to go over it with you 1-1.